packages/use-contractkit/src/contract-kit-provider.tsx

```diff
@@ -97,13 +97,16 @@ export const ContractKitProvider: React.FC<ContractKitProviderProps> = ({
   feeCurrency = CeloContract.GoldToken,
 }: ContractKitProviderProps) => {
   const isMountedRef = useIsMounted();
-  const previousConfig = useMemo(() => loadPreviousConfig(network), [network]);
+  const previousConfig = useMemo(
+    () => loadPreviousConfig(network, feeCurrency),
+    [network]
+  );
   const [state, _dispatch] = useReducer(contractKitReducer, {
     ...initialState,
     ...previousConfig,
     network: previousConfig.network || network,
+    feeCurrency: previousConfig.feeCurrency || feeCurrency,
```

Comment:
hmm i could see this being potentially confusing if a fee currency is passed in explicitly but the previous one is used? 

why this order and not `feeCurrency || previousConfg` ?

Reply:
Because then there is no purpose in saving the fee currency in local storage.